src/backends/interflop-cancellation/interflop_cancellation.c

```diff
@@ -101,7 +98,7 @@ static inline double _noise_binary64(const int exp, rng_state_t *rng_state) {
       _init_rng_state_struct(&RNG_STATE, ((t_context *)CTX)->choose_seed,      \
                              (unsigned long long)(((t_context *)CTX)->seed),   \
                              false);                                           \
-      *Z = *Z + _noise_binary64(e_n, &RNG_STATE);                              \
+      *Z += _noise_binary64(e_n, &RNG_STATE);                                  \
```

Comment:
I am ok since it is functionally eq, but not like it much when doing operations beyond simple integer induction variables.

Reply:
As you like, it's a matter of taste